src/main/java/net/binis/codegen/validation/sanitizer/NullAwareLambdaSanitizer.java

```diff
@@ -0,0 +1,17 @@
+package net.binis.codegen.validation.sanitizer;
+
+import java.util.function.Function;
+
+import static java.util.Objects.nonNull;
+
```

Comment:
Hey @binis2,

Do we need the below annotation here as well?
`@AsCode("((java.util.function.Function<{type}, {type}>) %s)")`

Not sure how the code gen parses those classes...